### ISSUE TITLE
Empty response causes hang in client response

### ISSUE THREAD

**Issue by bwehrle** [vlingo/vlingo-http#18]
When creating a Resource class, if the Response Body is empty, the HTTP client will hang.
```java
 public Completes<Response> noOpSuccess(final String param) {
        return Completes.withSuccess(
                Response.of(Created,
                        headers(of(Location,"/")), Body.from("")));
    }
```
This can be seen in the following branch using the example code in CartResourceShould class in the "vlingo-examples/vlingo-ecommerce" sample project I was working on.  I was looking at the http code to find a solution, but I haven't found the issue yet.

https://github.com/bwehrle/vlingo-examples/commit/6772b2cad3f937e641da25ab5f6b8315383b8eb3

**Comment by VaughnVernon** [comment]
There are variations of the `Response#of(...)` method found here are they, and they are tested and otherwise used in one form or another in several places:

https://github.com/vlingo/vlingo-http/blob/ed8ce0b17e14f272d1671cdafb927f7c008edacd/src/main/java/io/vlingo/http/Response.java#L16

These all produce an empt `Body`. A few such are:

https://github.com/vlingo/vlingo-http/blob/ed8ce0b17e14f272d1671cdafb927f7c008edacd/src/main/java/io/vlingo/http/resource/sse/SseStreamResource.java#L57

https://github.com/vlingo/vlingo-http/blob/ed8ce0b17e14f272d1671cdafb927f7c008edacd/src/main/java/io/vlingo/http/resource/sse/SseStreamResource.java#L66

Perhaps the issue is on your client consumer.

**Comment by bwehrle** [comment]
Hi @VaughnVernon ,
In this case my test code is behaving in the same way the "curl" BSD unix command is behaving, which was how I started testing my code.  Therefore, there is likely something else going on.  Now, the body is arguably being sent, since it is empty, but the connection never closed by the server, which it should be.  Perhaps this is working well for SSE clients?  I'll look at both examples more closely.

**Comment by VaughnVernon** [comment]
@aleixmorgadas @brian-wehrle-roche Is this related to #14 and could be marked closed? Did #14 fix this issue, Brian?

**Comment by bwehrle** [comment]
Hi I'm going to be  using my non-work account  from now on. (@bwehrle )-  I can't  repro this  now, so I vote  to close  this issue.
